recipes/multiprocessing-logging/meta.yaml

```
@@ -0,0 +1,39 @@
+{% set name = "multiprocessing-logging" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/jruere/multiprocessing-logging/archive/1b80bd458bec4bcf0f312c5fbdf1088fef7235fb.zip"
+  sha256: 4d67a0200fabede309cbe2f75708a70b8cfd8d63c6d6c5c42f469053b20a4cf6
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  noarch: python
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - multiprocessing_logging
```

Can we try to add ``pip check`` here please?
```suggestion
    - multiprocessing_logging
  requires:
    - pip
  commands:
    - pip check
```

---

Added